R/check_time_variables.R

```diff
@@ -91,7 +92,8 @@ check_time_variables<-function(data){
     dplyr::mutate(chk_TAD=case_when(
       !is.na(TAD) ~1,
       is.na(TSFM) & is.na(TAD)~1,
-      suppressWarnings(min(TSFM[EVID %in% 1]) %in% NA_integer_) & is.na(TAD) ~1,
+      suppressWarnings(min(TSFM[EVID %in% 1]) %in% NA_integer_ & is.na(TAD)) ~1,
+      suppressWarnings(min(TSFM[EVID %in% 1]) %in% "Inf" & is.na(TAD))~1,
```

Comment:
Why are there warnings for these two lines?  Can we remove suppressWarnings() or otherwise prevent the warnings?  (I generally worry that `suppressWarnings()` may hide something that we want to see.)

Reply:
Below warning message appears when attempting to find the minimum value of a vector of length zero. eg. min(TSFM[EVID %in% 1]) results in numeric(0):
_In min(data) : no non-missing arguments to min; returning Inf._

Created function min_no_warning() to address the above warning issue and added code lines for vectors with min values in NA or Inf, removed suppressWarnings(): 
   **is.na**(min_no_warning(var1[EVID` %in% 1])) & is.na(var2) ~1
   min_no_warning(var1[EVID %in% 1]) %in% **Inf** & is.na(var2)~1

05c7f00 Resolved issue related to warning for check_time_variables.R

Added variables STUDYID,USUBJID,EVID to check_var

Reply:
@SBcytel Thanks!  That's exactly the way that I was hoping a fix would be made (though I didn't know the warning that was being covered).  Now, we see warnings if there is a reason we should, but we don't worry about the zero-length case.